### PR TITLE
fix(select): list scrollbar height and timing (#6987)

### PR DESCRIPTION
**Pull request by selicens**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to main branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](./pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by wsxe9988** [comment]
@tangjinzhou 太久没有review。  不少人提了pr,及时review 发版么

File: components/vc-virtual-list/List.tsx [hunk 1]
```diff
@@ -497,6 +502,7 @@ const List = defineComponent({
         }}
         class={mergedClassName}
         {...restProps}
+        onMousemove={debounce(mousemove, 500)}
```
Comment (tangjinzhou) [inline components/vc-virtual-list/List.tsx:505]: 这个不需要吧，像mac滚动后显示滚动条，也不会因为鼠标移动影响滚动条的显示隐藏

**Review by tangjinzhou — CHANGES_REQUESTED** [review]

File: components/vc-virtual-list/List.tsx (hunk 1 above)
Comment (selicens) [reply components/vc-virtual-list/List.tsx:505]: antd展开2s后隐藏滚动条，滚动或者鼠标重新移入会再次显示，antdv这边鼠标重新移入是不会出现滚动条，[#6987](https://github.com/vueComponent/ant-design-vue/issues/6987)

**Comment by tangjinzhou** [comment]
我做了改动

**Pull request merged**